packages/crawler/reports/2022-12-19_data-aggregated.json

```diff
@@ -0,0 +1,1400 @@
+{
+    "crawledWebsites": [
+        "https://www.porsche.com/germany",
+        "https://finder.porsche.com/de/de-DE",
+        "https://login.porsche.com/login/de/de_DE",
+        "https://shop.porsche.com/de/de-DE",
+        "https://www.porsche.com/swiss/de"
+    ],
+    "aggregatedConsumedTagNames": {
+        "tagNames": {
+            "p-button": {
+                "amount": 20,
+                "hostPdsComponent": 0,
+                "slot": 0,
```

Comment:
it's not correct that p-button has no slots

Reply:
- rename "slot" to "children"

- change the logic of getting slot contents: get children instead of going into shadowDOM

Reply:
done